content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -582,6 +583,25 @@ This amendment fixes a bug in the [NonFungibleTokensV1][] amendment code where N
 This amendment has no effect unless the [NonFungibleTokensV1][] amendment is enabled. This amendment is obsolete because its effects are included as part of [NonFungibleTokensV1_1][].
 
 
+## fixNFTokenRemint
+[fixNFTokenRemint]: #fixnftokenremint
+
+| Amendment    | fixNFTokenNegOffer |
```

Comment:
Copy pasta'd from somewhere else by accident?

Reply:
That is definitely some sloppy linguine there. Fixed.